tests/fixtures.py

```diff
@@ -37,6 +37,11 @@ def do_nothing():
     """The best job in the world."""
     pass
 
+def raise_exc():
+    raise Exception('raise_exc error')
+
+def raise_exc_mock():
```

Comment:
Why do we need `raise_exc_mock`? Can't we just use `raise_exc()` as mock?

Reply:
> new_callable allows you to specify a different class, or **callable object, that will be called to create the new object**. By default AsyncMock is used for async functions and MagicMock for the rest.

from https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch

otherwise `raise_exc` is called and tests fails always since it is the same as write `raise Exception`.